lib/_fields.js

```diff
@@ -137,7 +137,11 @@ var fieldDefs = {
   url: {
     type: String,
     regEx: SimpleSchema.RegEx.Url,
-    max: 200
+    /* In AH-802 we had an uexpected error due to the 200 character limit.
```

Comment:
"unexpected". Sorry!

Reply:
I'm trying to be consistent with [umount](https://unix.stackexchange.com/questions/9832/why-is-umount-not-spelled-unmount)

Reply:
(No I'm not.  Good catch.)